python/amici/petab_objective.py

```diff
@@ -55,6 +60,10 @@ def simulate_petab(
 ) -> Dict[str, Any]:
     """Simulate PEtab model.
 
+    NB: supplied parameters are scaled internally, independently of
```

Comment:
something like...?

```suggestion
.. note::
    Supplied parameters are scaled internally according to `parameterScale` in the parameter table, independently of
```

Reply:
Clarify: What you mean is: Regardless of `scaled_parameters`, sensitivities will always be on the scale as specified in `parameterScale` in the parameter table?